openjdk/mmtkMutator.cpp

```diff
@@ -1,12 +1,19 @@
 
 #include "mmtk.h"
 #include "mmtkMutator.hpp"
+#include <atomic>
```

Comment:
Where is this used?

Reply:
Ah. I was doing some instrumentation with atomic integers. I will the include. Sorry.